### DISCUSSION TITLE
Smoothing noisy and oversampled RHI fields

### DISCUSSION THREAD

**Discussion by Ted-Whittock** [ARM-DOE/pyart#1289] [Q&A] [answered]
Hello,

I have a lot of RHI files with high vertical resolutions. We sampled every 0.2 degrees of elevation along the RHI's azimuth (sample file linked here: https://drive.google.com/drive/folders/19cQ3aS5qBt2gsQYjZu5cg14U_j7qt5J8?usp=sharing). As a result, we believe we have oversampled, and fields such as ZDR appear too noisy to be meaningfully interpreted. I have attached a sample image to demonstrate the noise. 

Is there an easy way (e.g., a built-in function, arguments within functions, etc.) to smooth the fields in the RHI file for plotting? Say, plotting an average or median of multiple rays instead of each individual ray, or any other methods? 

Thanks,
Ted

![RHI_3panel_2angle_DBZHCandZDRCandCC_20170203_160223_to_20170203_160333_Azimuth60 00_and_240 00](https://user-images.githubusercontent.com/107871515/195415211-ebddf1e6-c033-4401-830c-0354be3120c6.png)

**Comment by mgrover1** [comment, marked as answer]
Great question @Ted-Whittock - I would encourage you to check out our example contouring/adding a gaussian filter to smooth the data
[Contour example with smoothing](https://arm-doe.github.io/pyart/examples/plotting/plot_rhi_contours_differential_reflectivity.html#sphx-glr-examples-plotting-plot-rhi-contours-differential-reflectivity-py)

I would also encourage you to post this on our new Open Radar Discourse, which is a place for these sorts of questions! 
[Open Radar Discourse](https://openradar.discourse.group/)
